src/server.py

```diff
@@ -21,23 +37,61 @@ async def api_v1():
 async def api_v1_deal():
     return {"rank": random.choice(RANKS), "suit": random.choice(SUITS)}
 
-
+#creates new deck, adds deck ID and associated deck object to dictionary, generates new expiration time associated with it
 @app.post("/api/v2/deck/new")
 async def api_v2_deck_new():
     d = Deck()
-    return {"message": "deck created!"}
-
+    deck_id = generate_unique_deck_id()
+    deck_objects[deck_id] = d
+    expiration_time = time.mktime((datetime.now() + timedelta(hours=24)).timetuple()) #converts datetime to unix timestamp
+    expiration_times[deck_id] = expiration_time
+    hand = d.draw(5)  
+    return {
+            "deck_id": deck_id,
+            "expires": expiration_time,
```

Comment:
Ah, I suppose you can manage the expiration on the UI this way. If the deck is expired, then show an error if the user tries to draw after that time. It's tricky, but definitely makes things more robust.